src/iop/expfusion.c

```diff
@@ -1292,21 +1086,8 @@ static void _buil_weight_map(const float *const img_src, const size_t wd, const
     {
       const float *const rgb = img_src + (y * wd * ch) + x * ch;
 
-      float E = 1.f;
-      if(grey_projector != DT_PROJECTOR_NONE)
-      {
-        float lum = _grey_projector(rgb, grey_projector, work_profile);
-        if(fusion_colorspace == DT_FUSION_COLORSPACE_LAB) lum = powf(lum, exposure_optimum);
-        E = _well_exposedness(lum, weight_mode, exposure_optimum, exposure_width, exposure_left_cutoff, exposure_right_cutoff);
-      }
-      else
-      {
-        E = _well_exposedness(rgb[0], weight_mode, exposure_optimum, exposure_width, exposure_left_cutoff, exposure_right_cutoff) *
-              _well_exposedness(rgb[1], weight_mode, exposure_optimum, exposure_width, exposure_left_cutoff, exposure_right_cutoff) *
-              _well_exposedness(rgb[2], weight_mode, exposure_optimum, exposure_width, exposure_left_cutoff, exposure_right_cutoff);
-      }
-
-      img_map[y * wd + x] = E;
+      const float lum = powf(_grey_projector(rgb, grey_projector, work_profile), exposure_optimum);
```

Comment:
Are you sure it's a good idea to have the gamma function be the same as exposure optimum here?

If optimum is around 0.4-0.5 or so there wont' be much difference, but outside of that range things might get strange.